bool tracked
bool interested
bool indecisive
bool not_interested
bool stationary
float64 speed
float64 distance
geometry_msgs/Point position